Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -542,13 +563,33 @@ public void onGlobalLayout() {
                     int lineTop = layout.getLineTop(layout.getLineForOffset(matchLocation));
                     ((NestedScrollView) mRootView).smoothScrollTo(0, lineTop);
                     mShouldScrollToSearchMatch = false;
+                } else if (mNote != null && mNote.getSimperiumKey() != null) {
+                    ((NestedScrollView) mRootView).scrollTo(0, mPreferences.getInt(mNote.getSimperiumKey(), 0));
+                    mRootView.setOnScrollChangeListener(
+                        new View.OnScrollChangeListener() {
+                            @Override
+                            public void onScrollChange(View v, int scrollX, int scrollY, int oldScrollX, int oldScrollY) {
+                                mPreferences.edit().putInt(mNote.getSimperiumKey(), scrollY).apply();
+                            }
+                        }
+                    );
                 }
             }
         });
         setHasOptionsMenu(true);
         return mRootView;
     }
 
+    public void removeScrollListener() {
+        mRootView.setOnScrollChangeListener(
```

Comment:
Can this be simply:

```
mRootView.setOnScrollChangeListener(null);
```

Reply:
I thought I tried that and `setOnScrollChangeListener` didn't like `null` for some reason.  It seems to work now.  I updated the `removeScrollListener` method in .

Reply:
I forgot to push the changes.  The commit should be added now.